### PR TITLE
Timeline: limit items size

### PR DESCRIPTION
**Pull request by AdrienClairembault**
There is currently no limits to the size of timeline items:

image

This isn't recommended because long lines are hard to follow and read properly.

After adding a 700px hard limit (almost 90 chars max, which is still quite generous):

image

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |

### CODE REVIEW

File: css/includes/components/itilobject/_timeline.scss [hunk 1]
```diff
@@ -99,7 +99,7 @@
     .timeline-content {
         position: relative;
         border-top-left-radius: 0;
-        max-width: 100%;
+        max-width: 700px;
```
Comment (cedric-anne) [inline css/includes/components/itilobject/_timeline.scss:102]: Not sure it will work in all window resolution contexts.

Comment (cedric-anne) [reply css/includes/components/itilobject/_timeline.scss:102]: I am just dubious about rendering on a small resolution, since 700px can be more than 100% of the container.

Comment (AdrienClairembault) [reply css/includes/components/itilobject/_timeline.scss:102]: There is no `width` or `min-width` defined so small containers are not impacted.

**Review by orthagh — APPROVED** [review]

File: css/includes/components/itilobject/_timeline.scss (hunk 1 above)
Comment (cedric-anne) [reply css/includes/components/itilobject/_timeline.scss:102]: I thought we may have an overflow, but I am probably wrong then.

**Review by cedric-anne — APPROVED** [review]
